src/app/infrastructure/DragDrop/dropTargetFactory.tsx

```diff
@@ -41,6 +47,24 @@ export default function dropTargetFactory<TDrag, TDrop>(itemTypes: string | stri
         computeStyle?: (props: { isActive?: boolean, canDrop?: boolean, isOver?: boolean }) => CSSProperties;
     }
 
+    function computeStyleDefault({ isActive, canDrop, isOver }: DragDropStatus) {
+
+        let borderColor: string = 'transparent'
+        //let dropStatus = "";
```

Comment:
can this be removed?

Reply:
Cleaned this up a little more. to match a different way that I had done this.